metashare/api/gh.py

```diff
@@ -138,3 +142,32 @@ def try_to_make_branch(repository, *, new_branch, base_branch):
                 counter += 1
             else:
                 raise
+
+
+def create_updates_hook_for_repo(*, user, repository):
+    if settings.GITHUB_HOOK_ROOT_URL is None:
+        return
+
+    url = settings.GITHUB_HOOK_ROOT_URL + reverse(
+        "repository-hook", kwargs={"pk": str(repository.pk)}
+    )
+    secret = binascii.hexlify(os.urandom(24)).decode("utf-8")
+    repo = get_repo_info(user=user, repo_id=repository.repo_id)
+    repo.create_hook(
+        "web",
+        {"url": url, "content_type": "json", "secret": secret},
+        events=["push", "pull_request"],
+    )
+    repository.hook_secret = secret
```

Comment:
@wlonk We should definitely not be creating a separate webhook for each repository. With a GitHub App, we can configure a webhook once for the app, and then it will receive events for every repository where the app has been installed.

Reply:
I agree. This is a stop-gap until we do that.

Reply:
What are we waiting for?

Reply:
We can get a github app set up and start using it for the webhooks even if we aren't using it for oauth or API calls yet.

Reply:
Nothing except to do it. It's in-queue now.